data/README.md

```diff
@@ -1,6 +1,13 @@
 # Story Squad Data
 
-- Children's story submissions are not able to be made public due to [COPPA](https://www.ecfr.gov/cgi-bin/text-idx?SID=4939e77c77a1a1a08c1cbf905fc4b409&node=16%3A1.0.1.3.36&rgn=div5) guidelines, and our team decided to extend that to transcriptions as well out of precaution. 
+- According to https://www.ftc.gov/tips-advice/business-center/guidance/complying-coppa-frequently-asked-questions-0#F.%20Photos, FAQ F.2 states:
```

Comment:
This helps resolve all lingering issues regarding COPPA and children's right to privacy. It helps us in gathering data from this sample group. Great job.